12/alpine/Dockerfile

```diff
@@ -1,6 +1,6 @@
 FROM alpine:3.9
```

Comment:
Should this also bump to 3.10?

Reply:
We haven't bump the version in config, so I don't think so.

Reply:
It should, unless we've changed our policy

Reply:
Debian Buster was also released but not changed in the config yet, I suggest not to block nodejs release process.

Reply:
We get grief everytime we change os versions. I think the way we handle debian now is pretty solid so it isn't an issue anymore but until we use multiple versions of alpine, the switching version is going to be an issue.